packages/sdk-codegen-scripts/src/exampleMiner.ts

```diff
@@ -80,7 +80,17 @@ export const filterCodeFiles = (fileName: string) => {
   return ext in fileMiners
 }
 
-const skipFolder = (name: string, excludeList: string[] = ['node_modules']) =>
+export const IGNORE_PATHS = [
+  'node_modules',
+  'lib',
+  'dist',
+  'bazel-bin',
+  'build',
+  'bin',
+  '.build',
```

Comment:
.direnv, .github, .vscode, .idea, results?

Probably overkill

Reply:
Easy enough to add. I'll do that now.

Reply:
Added